### ISSUE TITLE
[FEATURE] Scrolling the canvas with the mouse.

### ISSUE THREAD

**Issue by magnetophon** [Chowdhury-DSP/BYOD#15]
Would it be possible to scroll over the canvas using the mouse-wheel and/or dragging the touchpad?

**Comment by jatinchowdhury18** [comment]
Definitely! I'm hoping to do an iOS version, so that will be a necessity.

**Comment by jatinchowdhury18** [comment]
With the new way that the UI is configured, especially how it's handling zoom in/out, I'm not 100% sure that scrolling will be completely necessary. I'd be curious to hear any differing opinions?

**Comment by magnetophon** [comment]
I just bult the latest version, and I'm not sure I understand what you mean.
How does the zooming alleviate the need for scrolling?

**Comment by jatinchowdhury18** [comment]
Yeah sorry, I really didn't explain that at all. Basically, with the new zoom functionality, it's not really "zooming in" on one part of the canvas, rather, the zoom functionality makes the module UIs larger. I think I like having it work this way, since it simplifies the controls, and since no modules will ever get "lost" off-screen. Plus the module that you've been working with most recently will be on top, and will gradually take up more of the screen as you zoom in.

With that in mind, I'm not really sure how the scrolling would work, given that you're always seeing the entire canvas. Maybe there's something I'm missing? Or maybe there's better reasons why the zoom functionality should work differently?

**Comment by magnetophon** [comment]
Thanks for explaining.
I see your point, and agree.

I'll leave the issue open so others aren't discouraged from reacting.
